### PR TITLE
Change attributes of `InterviewStatus`

### LINKED ISSUE #107: Change statuses in `InterviewStatus`
Currently `InterviewStatus` has 
- Not interviewed
- Interviewing
- Pending

We will change it to 
- Not scheduled (default) 
- Scheduled
- Completed

### PR DESCRIPTION
**Pull request by tiewweijian**
Changed InterviewStatuses from
- Interviewing
- Not Interviewed
- Pending 

To
- Not Scheduled
- Scheduled
- Completed

Closes #107

### CODE REVIEW

**Review by LeeEnEn — APPROVED** [review]
Apart from that line, everything else looks good to me! 🥂

File: src/test/java/seedu/address/logic/parser/AddCommandParserTest.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
     @Test
     public void parse_allFieldsPresent_success() {
         Candidate expectedCandidate = new CandidateBuilder(BOB).withTags(VALID_TAG_FRIEND)
-                .withApplicationStatus("Pending").withInterviewStatus("Pending").build();
+                .withApplicationStatus("Pending").withInterviewStatus("Not Scheduled").build();
```
Comment (LeeEnEn) [inline src/test/java/seedu/address/logic/parser/AddCommandParserTest.java:51]: Perhaps the `Pending` here could be replaced with a constant like `VALID_APPLICATION_STATUS`? Same for the interview status 👀

Comment (tiewweijian) [reply src/test/java/seedu/address/logic/parser/AddCommandParserTest.java]: thanks for pointing it out @domlimm ! Made the necessary changes 👍🏻

Comment (domlimm) [reply src/test/java/seedu/address/logic/parser/AddCommandParserTest.java]: 😍

**Review by charmainehly — APPROVED** [review]
LGTM! Thanks Wei Jian!!

**Pull request merged**